### DISCUSSION TITLE
Unable to register Elastica Agent

### DISCUSSION THREAD

**Discussion by Pencat89** [Security-Onion-Solutions/securityonion#11095] [2.4]
Times out and does not provide an agent to accept
Already entered the CIDR for the network.
Using distributed mode. Separate Manager and Search node.
Installing the windows agent and using the script in powershell to install.

**Comment by defensivedepth** [comment]
Did you use the installer from the Dowloads page in the Security Onion Console?

https://docs.securityonion.net/en/2.4/elastic-agent.html#deployment
